src/Compilers/VisualBasic/Test/Semantic/Semantics/NonTrailingNamedArgumentsTests.vb

```diff
@@ -0,0 +1,258 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+Imports VB = Microsoft.CodeAnalysis.VisualBasic
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    <CompilerTrait(CompilerFeature.NonTrailingNamedArgs)>
+    Public Class NonTrailingNamedArgumentsTests
```

Comment:
>NonTrailingNamedArgumentsTests [](start = 17, length = 30)

It looks like there are no tests with named argument used for ParamArray parameter.